.github/workflows/test.yml

```diff
@@ -21,7 +21,10 @@ jobs:
           auto-activate-base: false
           show-channel-urls: true
           use-mamba: true
-
+      - name: Install local module mopi
+        shell: bash -l {0}
+        run: |
+          pip install -e .
```

Comment:
Now that ``mopi`` is a module we need to build it locally before running the tests on github actions.
**The reason:**
The packaging requires that the module is under ``src/`` directory.
-> So when running test we would need to refer to the mopi module as ``src.mopi`` and all other imports aswell
-> If we moved ``tests/`` under ``src/`` the discovery of tests has to be specified (which could be an alternate solution)